### PR TITLE
Fix #156 - Add policies

### LINKED ISSUE #156: Add Iubenda Cookie Policy and Privacy Policy
https://www.iubenda.com/en/

### PR DESCRIPTION
**Pull request by Psykopear**

### CODE REVIEW

File: django-website/frontend/client/scss/layout/_footer.scss [hunk 1]
```diff
@@ -112,3 +112,12 @@
...
 }
+
+.footer-policies {
+  display: inline-block;
+  margin-top: .382rem;
+  font-size: 0.8rem;
+  a {
+    text-decoration: underline;
+  }
+}
```
Comment (rub) [inline django-website/frontend/client/scss/layout/_footer.scss]: Add this declaration:
```
a + a {
  margin-left: .618rem
}

File: django-website/frontend/client/scss/layout/_footer.scss [hunk 2]
```diff
@@ -112,3 +112,12 @@
     margin-left: 1.618rem;
   }
 }
+
+.footer-policies {
+  display: inline-block;
+  margin-top: .382rem;
+  font-size: 0.8rem;
+  a {
```
Comment (rub) [inline django-website/frontend/client/scss/layout/_footer.scss:120]: Add a space between the last rule and the declaration

File: django-website/frontend/client/scss/layout/_footer.scss [hunk 3]
```diff
@@ -112,3 +112,12 @@
...
+    text-decoration: underline;
+  }
```
Comment (rub) [inline django-website/frontend/client/scss/layout/_footer.scss:122]: Add this declaration:
```
a + a {
  margin-left: .618rem
}
```

File: django-website/frontend/client/scss/layout/_footer.scss [hunk 4]
```diff
@@ -112,3 +112,12 @@
     margin-left: 1.618rem;
   }
 }
+
+.footer-policies {
+  display: inline-block;
```
Comment (rub) [inline django-website/frontend/client/scss/layout/_footer.scss]: Unneeded rule

**Review by rub — CHANGES_REQUESTED** [review]

**Review by synasius — APPROVED** [review]

**Pull request merged**